dashboard/app/helpers/application_helper.rb

```diff
@@ -65,6 +65,8 @@ def best_activity_css_class(user_levels)
       'review_rejected'
     elsif result == Activity::REVIEW_ACCEPTED_RESULT
       'review_accepted'
+    elsif user_level.try(:locked)
+      'locked'
```

Comment:
Confirming that this is the correct precedence in this list — if a peer review level is ever lockable for some reason, the peer review status will take precedence over the locked status.